wireapp/wire-ios-mono
# fix: diverse logs - WPB-9939

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-9939" title="WPB-9939" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-9939</a>  Crash accessing selfUser in NSE - Beta
  </td></table>
  <br />
 













### Issue

While investigating a crash on the NSE, I found a couple of issues with the logs we have, here's some fixes:

* redact emoji and messageId in generic message log
* include log level in messages for file logger
* log corecrypto teardown error
* move SafeFileContext log to WireLogger
* fix build_version in extension

### Testing

* run app
* send logs via debug report
* check logs content

---

### Checklist
